ratatoskr/app/urls.py

```diff
@@ -14,6 +14,7 @@
     path('schedule/<schedule:schedule>/create-timeslots', views.create_timeslots, name='create-timeslots'),
     path('schedule/<schedule:schedule>/<datetime:date>/reserve/<timeslot:timeslot>', views.reserve_timeslot, name='reserve-timeslot'),
     path('schedule/<schedule:schedule>/<datetime:date>/view/<timeslot:timeslot>', views.view_reservations, name='view-reservations'),
+    path('confirm/<int:reservation>', views.confirm_reservation, name='confirm-reservation'),
```

Comment:
You should probably use the new model URL converters. We have them for `Schedule`, `TimeSlot`, and `Reservation`. These URL converters allow us to eliminate that one line of code used for fetching the data from the database. You can look at how these converters are implemented in the `app.converters` module. 

In this case here, you would change the `<int:reservation>` to `<reservation:reservation>` to take advantage of the URL converter.